docs/pages/desktop-access/getting-started.mdx

```diff
@@ -314,6 +314,61 @@ Computer Configuration > Policies > Administrative Templates > Windows Component
   ![Disable Require](../../img/desktop-access/disable.png)
 </Figure>
 
+### Configure certificate for RDP connections
+
+<Admonition type="note" title="Secure Cipher Suites">
+  Teleport's RDP client supports only secure algorithms
+  for making TLS connections, so we have to configure a Windows server
+  to support those cipher suites as well.
+  This step is only *neccessary* for Windows Server 2012 R2 as it does not support
+  secure algorithms by default.
+</Admonition>
+
+
+In this step we'll create a new certificate template that uses
+elliptic curve cryptography, and then configure our GPO
+to use the newly created template to issue certificates used for Remote Desktop connections.
+
+# Create a certificate template
+
+1. Begin by navigating to:
+```
+Start > Control Panel > Administrative Tools > Certificate Authority
+```
+to open the CA Microsoft Management Console (MMC) GUI.
+2. Open your CA computer and right-click on `Certificate Templates`, then select `Manage`.
+3. Find `Computer` template on the list, right-click on it, then select `Duplicate Template`.
+4. In the `Compatibility` tab change `Certification Authority` to `Windows Server 2012 R2` and click `OK`.
+In the same tab change `Certificate recipient` to `Windows Server 2012 R2` and click `OK`.
+5. Go to `General` tab and change `Template display name` to `RemoteDesktopAccess`. Make sure `Template name` is also `RemoteDesktopAccess`.
+6. In the `Cryptography` tab change `Provider Category` to `Key Storage Provider`, then `Algorithm name` to `ECDH_P384`. Also, change `Request hash` to `SHA384`.
```

Comment:
Does it have to be P384/SHA384 or did you pick those because they are the best available? Is there a lower bound?

No need to change the text, I'm just curious. Although, if there is a lower bound, let's mention it in the description that precedes this (ie, "certificates must be P384 or better").

Reply:
Yeah, it have to be this one. I picked it as it is enabled be default by Windows Server 2012 R2 and is also supported by rustls library. The other one that would also work is P521/SHA384 but it would require user to enable it manually (few extra steps that doesn't provide much of difference in our case).